codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Stream.java

```diff
@@ -44,6 +68,11 @@
     State state();
 
     /**
+     * Gets the endpoint that created this stream.
+     */
+    Http2Connection.Endpoint<? extends Http2FlowController> createdBy();
```

Comment:
Does this have to be exposed on the interface? It looks like it is only used in `DefaultHttp2Connection`. Users could always check `local().created(...)` or `remote().created(..)`. WDYT?